### PR TITLE
Added devtron to list.md 

### PR DESCRIPTION
**Pull request by AbhiiVops**
Corrected some of the mistakes in Readme.md

Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+
+- **Swag**: 1st Rank : Devtron's Limited Edition Swag Kit
+          2nd Rank : Devtron's Limited Edition Diary + stickers
+          3rd Rank : Devtron's Limited Edition stickers
+- **Requirements**:  Star the [Devtron](https://github.com/devtron-labs/devtron) repo on Github
+                     Follow us on [Twitter](https://twitter.com/DevtronL)
+                     Make some art and share it on Twitter. Tag @DevtronL and use #Hacktoberfest, #Devtron on your post.
+- **How to sign up**: Open this [Github Repo](https://github.com/devtron-labs/devtron) ,Fork it and star it . When submitting a PR please add Hacktoberfest in the PR Description. This would give us a confirmation on whether the PR is for Hacktoberfest.
+- **Notes**: Read the contributions guidelines caredfully and for such opportunites in devtron [Click Here](https://github.com/devtron-labs/hacktoberfest2022/blob/main/README.md#book-contribution-guidelines)
+
```
Comment (crweiner) [inline docs/list.md:210]: Thank you for adding Devtron to the Swag List! I'd encoruage you to look at the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing/) to make sure you're formatting things correctly (as well as other entries on the List). 

Netlify will also provide you with a build preview with each new commit, so feel free to use those previews to make sure that things look the way you want them to.

File: Readme.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 
 ![Hacktoberfest logo](docs/img/Hfest-Logo-2-Color-Manga.png)
 
-Hello all you beautiful nerds and welcome to another year of [Hacktoberfest](https://hacktoberfest.com/)! at the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com)
```
Comment (crweiner) [inline Readme.md:5]: I think that all of these proposed changes to the Readme file are of low value. I'd much rather appreciate you adding an organization to the Swag List from one of the Issues. Please un-do these propsed chages to the Readme you made; I will not be able to merge any _good contribution_ with this still in place.

**Review by crweiner — CHANGES_REQUESTED** [review]
Hi @AbhiiVops! Thank you for your PR to the Hacktoberfest Swag List! A few things to address below, please. 

Ping me when you're ready for me to take another look! 👀

**Comment by AbhiiVops** [comment]
> 

I have made the suggested changes. Please check once

**Comment by AbhiiVops** [comment]
> Hi @AbhiiVops! Thank you for your PR to the Hacktoberfest Swag List! A few things to address below, please.
> 
> Ping me when you're ready for me to take another look! 👀

I have made the changes. Please have a look at it

File: docs/list.md [hunk 3]
```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/) so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/)
 
+#### **devtron**
```
Comment (crweiner) [inline docs/list.md:200]: I started to make the changes myself to this entry, but there were so many that I stopped. I would like you to please review your PR and format this Devtron entry differently. 

Devtron appears to be rewarding people for meaningful PRs during their several different events. This PR appears to only capture their Art-A-Thon event, not anything else.

File: docs/list.md [hunk 4]
```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+
+- **Swag**: Devtron's Limited Edition Swag Kit for 1st Rank , Devtron's Limited Edition Diary + stickers for 2nd Rank,  Devtron's Limited Edition stickers for 3rd Rank
+- **Requirements**:
```
Comment (crweiner) [inline docs/list.md:203]: Please list the requirements of each event, summarized (do not just copy and paste from their Readme file...).

File: docs/list.md [hunk 5]
```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+    - Star the [Devtron](https://github.com/devtron-labs/devtron) repo on Github
+    - Follow us on [Twitter](https://twitter.com/DevtronL)
+    - Make some art and share it on Twitter. Tag @DevtronL and use    #Hacktoberfest, #Devtron on your post.
+- **How to sign up**: Open this [Github Repo](https://github.com/devtron-labs/devtron) ,Fork it and star it . When submitting a PR please add Hacktoberfest in the PR Description. This would give us a confirmation on whether the PR is for Hacktoberfest.
```
Comment (crweiner) [inline docs/list.md:207]: Please double check all formatting, spelling, punctuation, etc.

File: docs/list.md [hunk 6]
```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Issues**: Checkout [Devtron Repo](https://github.com/devtron-labs/hacktoberfest2022/blob/main/README.md#book-contribution-guidelines).
```
Comment (crweiner) [inline docs/list.md:208]: The Issues section, if included (as it's optional), is when an organization wants to highlight certain GitHub Issues as counting for Hacktoberfest. If that is not the case, then it should not be included here.

File: docs/list.md (hunk 3 above)
Comment (crweiner) [inline docs/list.md:200]:
```suggestion
#### **Devtron**
```

File: docs/list.md [hunk 7]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [learn-graphql issues](https://github.com/hasura/learn-graphql/issues) and [graphql-engine issues](https://github.com/hasura/graphql-engine/issues).
 - **Notes**: For more information, check out [this blog post](https://hasura.io/blog/hasura-joins-hacktoberfest-3rd-year-in-a-row/) and [this blog post too](https://hasura.io/blog/with-hasura-celebrate-open-source-in-style-with-hacktoberfest/).
 
+#### **hacksquad**
```
Comment (crweiner) [inline docs/list.md:265]:
```suggestion
#### **HackSquad**
```

File: docs/list.md [hunk 8]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+
+- **Swag**: T-shirts, hacksquad printed cups,carry bags,stickers,Hoodie
```
Comment (crweiner) [inline docs/list.md:267]: Please check your punctuation, spacing, spelling, etc.

File: docs/list.md [hunk 9]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Requirements**:  Register to the HackSquad using your GitHub, Join a squad or get assigned to a random squad, contribute code and get Swag!
```
Comment (crweiner) [inline docs/list.md:268]: Don't just copy and place, please. And again, check punctuation and spelling, please.

File: docs/list.md [hunk 10]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **How to sign up**: Open this [Hacksquad](https://www.hacksquad.dev/) ,Click on join now and there you go.
```
Comment (crweiner) [inline docs/list.md:269]:
```suggestion
- **How to sign up**: [Sign up on the Hacksquad website](https://www.hacksquad.dev/).
```

File: docs/list.md [hunk 11]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Issues**: Checkout [goodfirstissue](https://goodfirstissue.dev/) and contribute to it.
```
Comment (crweiner) [inline docs/list.md:270]: See above note on the Issues section regarding Devtron; please remove Issues here as it's not relevant.

File: docs/list.md [hunk 12]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Notes**:
+    - The top 60 squads will win awesome swag! around ~300 winners!
```
Comment (crweiner) [inline docs/list.md:272]: How many people are on a squad?

File: docs/list.md [hunk 13]
```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+    - Each hour we calculate the number of MERGED PRs of each squad member and sum them all up. Each PR is worth 1 point. By the end of the event, the top 60 squads will win awesome swag!
```
Comment (crweiner) [inline docs/list.md:273]: "Each hour **_we calculate_**..." _We_ over here at the Hacktoberfest Swag List don't do any calculations! Again, please don't copy and paste, you need to re-write what they have to instead be relevant to the visitors of this site/repo.

File: docs/list.md [hunk 14]
```diff
@@ -251,6 +262,19 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Requirements**:
+    Register to the HackSquad using your GitHub, Join a squad or get assigned to a random squad, contribute code and get Swag!
+- **How to sign up**: Open this [Hacksquad](https://www.hacksquad.dev/) ,Click on join now and there you go.
+- **Issues**: Checkout [goodfirstissue](https://goodfirstissue.dev/) and contribute to it.
+- **Notes**:
+    - The top 60 squads will win awesome swag! around ~300 winners!
+    - Each hour we calculate the number of MERGED PRs of each squad member and sum them all up. Each PR is worth 1 point. By the end of the event, the top 60 squads will win awesome swag! 
+    - To win swag, even if your team wins, you would need to have 1 MERGED PR
+
+
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:275]: Please make the robot happy here.

File: docs/list.md [hunk 15]
```diff
@@ -251,6 +262,19 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+#### **Hacksquad**
+
+- **Swag**: T-shirts, hacksquad printed cups,carry bags,stickers,Hoodie
+- **Requirements**:
+    Register to the HackSquad using your GitHub, Join a squad or get assigned to a random squad, contribute code and get Swag!
+- **How to sign up**: Open this [Hacksquad](https://www.hacksquad.dev/) ,Click on join now and there you go.
+- **Issues**: Checkout [goodfirstissue](https://goodfirstissue.dev/) and contribute to it.
+- **Notes**:
+    - The top 60 squads will win awesome swag! around ~300 winners!
+    - Each hour we calculate the number of MERGED PRs of each squad member and sum them all up. Each PR is worth 1 point. By the end of the event, the top 60 squads will win awesome swag!
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:273]: Please make the robot happy here.

**Review by crweiner — CHANGES_REQUESTED** [review]
Hello @AbhiiVops, there are lots of required changes that need to take place before I can merge your contribution.

The biggest things are:

1. Please do not just copy and paste text from the site, as the wording is often all off.
2. Please check your spelling, punctuation, capitalization, etc.
3. Please re-read what the `-**Issues**:` line item is for.
4. Please include all contribution/swag opportunities offered by an organization, not just one.

Ping me again when you'd like another look, thank you!

**Comment by crweiner** [comment]
Devtron has been added in #662, please feel free to open a new PR to add HackSquad and follow the edits I suggested above! 👍

**Pull request closed without merging**